app/scripts/modules/openstack/src/help/openstack.help.ts

```diff
@@ -17,7 +17,9 @@ const helpContents: {[key: string]: string} = {
   'openstack.network.floatingip': '<p>Whether or not each instance in the server group should be assigned a floating ip.</p><p>Default: <b>No</b></p>',
   'openstack.network.floatpool': 'The network from which to allocate a floating ip',
   'openstack.serverGroup.userData': '<p>Provides a script that will run when each server group instance starts.</p>',
+  'openstack.serverGroup.availabilityZones': 'The zones which will be used to deploy the application. If left blank all zones will be utilized.',
```

Comment:
I think we need to declare somehow in the language that this piece requires Openstack's Senlin. Something like, "Creates a Senlin Zone Placement Policy that balances equally across the selected zones".
Also, if left blank, the behavior is based on a user's configuration of their Openstack, so we can't necessarily say that all zones will be used.

Reply:
Updated the Help Text!